hazelcast/src/main/java/com/hazelcast/spi/NodeEngine.java

```diff
@@ -287,4 +288,6 @@
      * @since 3.8
      */
     MemberVersion getVersion();
+
+    ClusterWideConfigurationService getConfigurationService();
```

Comment:
Shouldn't return `ConfigurationService` instead of `ClusterWideConfigurationService`? Exposing an internal service impl via spi.